test/studies/shootout/submitted/revcomp2.execopts

```diff
@@ -1 +1 @@
---n=0 < fasta-revcomp.small # revcomp.good
+--n=0 < fasta-revcomp.small # revcomp2.good
```

Comment:
I'd lean towards removing the `# revcomp.good` portion here (and potentially the original .good file itself, since I don't think anything would be using it after you're done).  I think we're more likely to not look at this file again when the warnings are resolved and only update `revcomp2.good` and `revcomp3.good`, rather than remembering this exact PR and reverting it, which would leave a stranded .good file and this no longer necessary specification of which .good file to use.

However, as you've done it today is perfectly valid and good, and I wouldn't object to merging it as is :) Thanks and sorry for missing it in the original review!

Reply:
Ok, I've removed the old `.good` file and the unneeded execopts.

No worries, and thanks for pointing this out!